index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet">
+    <title>Portfolio: Mobile Version Skeleton</title>
+</head>
+<body>
+    <section class="mobile-skeleton">
+        <header class="header">
+            <nav class="navbar">
+                <ul class="header-container">
+                    <li class="logo"><h4>My Logo</h4></li>
```

Comment:
- [ optional ] you can put your name in place of `my logo` since you don't have a logo yet.